packages/template-retail-react-app/README.md

```diff
@@ -43,15 +43,15 @@ The configuration features include:
 
 ### Dedicated Configuration Files
 
-At project generation a single `default` configuration is created. For most cases a single configuration file will get the job done. But there are times where you want to have a different configuration for a different environment. These environments for example could be, a developers local machine, a remote `production` environment, or a remote `staging` environment. In all these cases you want to use a different configuration for each. 
+At project generation a single `default` configuration is created. For most cases a single configuration file will get the job done. But there are times when you want to have a different configuration for a different environment. These environments for example could be, a developers local machine, a remote `production` environment, or a remote `staging` environment. In all these cases you want to use a different configuration for each. 
 
 When multiple configurations are present, which configuration file to used is resolved in the following way: 
 
 1. Check to see if there is a config file named after your remote Managed Runtime environment and use this if one is present.
 2. If we are running on a developer machine, look for a `local` configuration and use this if one is present.
 3. Load the `default` config file `config/default.js`.
 
-This allows you do these but not limited to things:
+This allows you to do these but are not limited to things:
 
 - Each developer can have their own configuration, connection to their own Commerce API sandboxes.
 - Deploy a single codebase application to multiple environments with their own specific configurations (Multiple B2C Site with Different Domains).
```

Comment:
```suggestion
- A single PWA Kit codebase can be deployed to multiple environments, each with their own configuration.
```

Reply:
Previously we have `This configuration system allows you to:`, it makes sense to start this sentence with `deploy...`

Reply:
Sure, we can do that but we need to make both list items parallel (consistent sentence structure).

Reply:
Anw, let's not too caught up with this section here. We can always fix it up later on. I will stop pushing any change and wait for the CI to merge this PR . Thank you for all the feedback @rsexton404!

Reply:
Scratch that. I'm just going to use the text we already wrote for the docs instead.